isEmpty.js

```diff
@@ -27,11 +19,38 @@ const hasOwnProperty = Object.prototype.hasOwnProperty
  * isEmpty(null)
  * // => true
  *
- * isEmpty(true)
+ *  * isEmpty(undefined)
+ * // => true
+ *
+ * isEmpty(false)
  * // => true
  *
  * isEmpty(1)
  * // => true
+ * 
+ * isEmpty({})
+ * // => true
```

Comment:
nice! this is most important example to shows the cool point of `_.isEmpty()`, because of this: (javascript)

```javascript
  isEmpty({});               // false
```

```javascript
  _.isEmpty({});               // true
```

I think it's would be great to add this example to [documentation on sait](https://lodash.com/docs/4.17.15#isEmpty) too

Reply:
@jdalton Can you please review my changes?